MdePkg/Test/Mock/Library/GoogleTest/MockSafeIntLib/MockSafeIntLib.cpp

```diff
@@ -0,0 +1,116 @@
+/** @file MockSafeIntLib.cpp

```

Comment:
edk2 repo does not put the file name in doxygen @file statements.  Removes the need to update that content if file is renamed.

Reply:
Done